### DISCUSSION TITLE
data_custome_control

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#63] [Announcements]
# data_custome_control

条件结构

https://lv.qizhen.xyz/data_custome_control

**Comment by XXWJ4Zzzz** [comment]
您好，本章中第二节“用户自定义控件”的小节“自定义类型”被错打为“自定义类行”。

**Reply by ruanqizhen** [reply]
已修改，多谢指正！

**Comment by tetsuoyyx** [comment]
阮老师，我定义了自定义类型（内容是一个包含下拉列表控件和布尔按钮的簇）以后，运行时点击其中的控件没有任何反应是为什么？

**Reply by ruanqizhen** [reply]
这个有很多可能性的，需要你把代码贴出来看一下才能确定。
不过，你可以先查看一些基本设置：控件是控制型的还是显示型的，程序运行是否有错误，程序是否被卡在哪里不动了等。
